packages/framework/fluid-static/src/types.ts

```diff
@@ -69,6 +69,43 @@ export interface ContainerSchema {
  * Events that trigger when the roster of members in the Fluid session change.
  * Only changes that would be reflected in the returned map of IServiceAudience's getMembers method
  * will emit events.
+ *
+ * ### "membersChanged"
+ *
+ * The membersChanged event is emitted when a member is either added or removed.
+ *
+ * #### Listener signature
+ *
+ * ```typescript
+ * () => void;
+ * ```
+ *
+ * ### "memberAdded"
+ *
+ * The memberAdded event is emitted when a member joins the audience.
+ *
+ * #### Listener signature
+ *
+ * ```typescript
+ * (clientId: string, member: M) => void;
+ * ```
+ * - `clientId` - A unique identifier for the client
+ *
```

Comment:
The blank lines between params are a little ugly, but necessary to get the bulleted list to format properly.